eras/babbage/impl/testlib/Test/Cardano/Ledger/Babbage/Imp/UtxowSpec.hs

```diff
@@ -63,7 +64,8 @@ spec = describe "UTXOW" $ do
   it "ExtraRedeemers/RedeemerPointerPointsToNothing" $
     -- There is ExtraRedeemers test for PlutusV1 in Alonzo, thus we start with PlutusV2
     forM_ ([PlutusV2 .. eraMaxLanguage @era] :: [Language]) $ \lang -> do
-      logEntry $ "Testing for " ++ show lang
+      logEntry $ "Testing for " <> viaShow lang
+      logEntry $ "Testing for " <> viaShow lang
```

Comment:
A bit of duplication. I've also suggested keeping type signature of `logEntry` unchanged, so we wouldn't have to change this kind of occurrences
```suggestion
      logEntry $ "Testing for " <> show lang
```